src/clj/harja/palvelin/integraatiot/velho/varusteet.clj

```diff
@@ -64,57 +65,84 @@
   [:ominaisuudet :tyyppi])
 
 (def kaiteet
-  "tl501 Kaiteet" {:kohdeluokka "kaiteet" :palvelu "varusterekisteri" :api-versio "v1"
+  "tl501 Kaiteet" {:kohdeluokka "kaiteet" 
+                   :palvelu "varusterekisteri" :api-versio "v1"
                    :nimiavaruus "varusteet" :kohdeluokka->tyyppi-fn varuste-kohdeluokka->tyyppi
-                   :tyyppi-polku varuste-tyyppi-polku})
+                   :tyyppi-polku varuste-tyyppi-polku 
+                   :oid-prefix "1.2.246.578.4.3.1"
```

Comment:
Käytetäänkö näitä missään vai jäikö nämä turhiksi?

Reply:
Ne jäi kyllä nyt lähinnä dokumentaatioksi eli voisi kyllä tietty karsia pois

Reply:
Eihän niistä periaatteessa mitään haittaakaan kyllä ole, ja voi olla jopa ihan hyvä tietää jatkossa, että OID:eihin kuuluu ikään kuin metatietoa.